meta-balena-common/recipes-kernel/linux-firmware/linux-firmware_20210511.bbappend

```diff
@@ -0,0 +1 @@
+PSEUDO_IGNORE_PATHS_append = ",/run/"
```

Comment:
Where does this get used? Normally an append needs to be prefixed with a space. Is the comma required?
```
PSEUDO_IGNORE_PATHS_append = " ,/run/"
```

Reply:
Nevermind, I see this now.
https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?h=gatesgarth&id=4b94d45be2f4b46e3d46fb90050edc71175ab18c

Reply:
It turns out, this doesn't actually fix the issue. I didn't see it again after pushing that commit until now.

Reply:
Finally got some better output on this:
```
NOTE: recipe linux-firmware-1_20210511-r0: task do_package: Started
ERROR: linux-firmware-1_20210511-r0 do_package: Error executing a python function in exec_python_func() autogenerated:

The stack trace of python calls that resulted in this exception/failure was:
File: 'exec_python_func() autogenerated', lineno: 2, function: <module>
     0001:
 *** 0002:perform_packagecopy(d)
     0003:
File: '/work/build/../layers/poky/meta/classes/package.bbclass', lineno: 747, function: perform_packagecopy
     0743:        rpath_replace (dvar, d)
     0744:}
     0745:perform_packagecopy[cleandirs] = "${PKGD}"
     0746:perform_packagecopy[dirs] = "${PKGD}"
 *** 0747:
     0748:# We generate a master list of directories to process, we start by
     0749:# seeding this list with reasonable defaults, then load from
     0750:# the fs-perms.txt files
     0751:python fixup_perms () {
File: '/usr/lib/python3.6/subprocess.py', lineno: 356, function: check_output
     0352:        # empty string. That is maintained here for backwards compatibility.
     0353:        kwargs['input'] = '' if kwargs.get('universal_newlines', False) else b''
     0354:
     0355:    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
 *** 0356:               **kwargs).stdout
     0357:
     0358:
     0359:class CompletedProcess(object):
     0360:    """A process that has finished running.
File: '/usr/lib/python3.6/subprocess.py', lineno: 438, function: run
     0434:            raise
     0435:        retcode = process.poll()
     0436:        if check and retcode:
     0437:            raise CalledProcessError(retcode, process.args,
 *** 0438:                                     output=stdout, stderr=stderr)
     0439:    return CompletedProcess(process.args, retcode, stdout, stderr)
     0440:
     0441:
     0442:def list2cmdline(seq):
Exception: subprocess.CalledProcessError: Command 'tar -cf - -C /work/build/tmp/work/all-poky-linux/linux-firmware/1_20210511-r0/image -p -S . | tar -xf - -C /work/build/tmp/work/all-poky-linux/linux-firmware/1_20210511-r0/package' returned non-zero exit status 134.

Subprocess output:
abort()ing pseudo client by server request. See https://wiki.yoctoproject.org/wiki/Pseudo_Abort for more details on this.
Check logfile: /work/build/tmp/work/all-poky-linux/linux-firmware/1_20210511-r0/pseudo//pseudo.log
Aborted (core dumped)
```